code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm

```diff
@@ -0,0 +1,246 @@
+/mob/living/simple_animal/hostile/gorilla
+	name = "Gorilla"
+	desc = "A ground-dwelling, predominantly herbivorous ape that inhabits the forests of central Africa."
```

Comment:
What do you think about saying that they are from earth instead of central africa? Sounds too specific since only humans are from this planet.

Reply:
Sure